manifests/base/openshift/scc.yaml

```diff
@@ -0,0 +1,30 @@
+# scc for the Kepler
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: kepler-scc
+# To allow running privilegedContainers
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+allowHostNetwork: true
+allowHostPorts: true
+allowHostIPC: true
+allowHostPID: true
```

Comment:
Hey, as a best practice I would actually suggest restricting the kepler pods a little bit more as done lately in the kepler-operator as seen [here](https://github.com/sustainable-computing-io/kepler-operator/pull/185/files#diff-3bb56f572edbdbecf3c3fcf5d35887994d95caa410bbb18013f6e1e2838c9658R325).

I still need to raise a PR to also integrate this also into the main kepler repo but I think it would be good to already do this here.

Reply:
@BGrasnick Thank you so much for the review. Agree. Just update the scc file, please confirm.